playbooks/05-clients.yaml

```diff
@@ -20,6 +20,6 @@
   gather_facts: false
 
   tasks:
-    - name: Placeholder for client configuration.
-      ansible.builtin.debug:
-        msg: "Temporary placeholder"
+    - name: build neerali-fio container on clients
```

Comment:
Can we add a condition here instead of `fio/tasks/main.yaml`?

Reply:
Added  the condition here